simulator/base/px4/sitl.Dockerfile

```diff
@@ -6,8 +6,8 @@
 INCLUDE+ ./px4builder.Dockerfile
 
 # Add support for remote host in SITL & build it
-RUN git fetch --recurse-submodules=no origin fe7908feb0de5ee8a4465619098f09b987ab011d \
-    && git cherry-pick --no-commit fe7908feb0de5ee8a4465619098f09b987ab011d
+# RUN git fetch --recurse-submodules=no origin fe7908feb0de5ee8a4465619098f09b987ab011d \
+#     && git cherry-pick --no-commit fe7908feb0de5ee8a4465619098f09b987ab011d
```

Comment:
Remove if no longer necessary. Need to check carefully when not using network_mode=host

Reply:
Am at lab at the moment, if you have a minute you could verify on windows? Otherwise will verify tonight